public/js/main.js

```diff
@@ -167,8 +156,21 @@ bingo.controller('homeController', function($scope, $http, $location, bingosocke
 
 });
 
-bingo.controller('bingoController', function($scope, $document, $http, $routeParams, bingosockets) {
+bingo.controller('bingoController', function($scope, $document, $http, $location, $routeParams, bingosockets) {
   // Responsive bingo card: keep squares square.
+
+  //Initialize room information
+  $scope.players = []
+
+  $scope.host_name = $location.search().host.name;
```

Comment:
Why are we passing all this info through the URL when we have an initialization api call for getting data?
Also, we can't pass objects through urls. Did you observe urls like 
"http://localhost:3000/#/game/5534852cba3421c8301b7fdd?currUser=%5Bobject%20Object%5D&roomname=Alphabingo" ? I did.

No information is passed successfully to the bingo page.